src/adaptors/articles.ts

```diff
@@ -0,0 +1,13 @@
+import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
+
+export default interface Articles extends AsyncIterable<JsonLdObj> {
+    count(): Promise<number>;
+
+    get(id: Iri): Promise<JsonLdObj>;
+
+    set(article: JsonLdObj): Promise<void>;
+
+    delete(id: Iri): Promise<void>;
+
+    has(id: Iri): Promise<boolean>;
+}
```

Comment:
I'm getting an eslint error here:

```
/app/src/adaptors/articles.ts
  13:2  error  Missing semicolon  semi
```

Is this possibly a bug with the linter?

Reply:
My 2 cents would be that interfaces fall into the same realm as classes and functions (rather than expressions such as `{ ... }`) so they shouldn't require a semicolon. But the [rules I know](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) apply to JavaScript rather than TypeScript, so unclear.

Reply:
(https://github.com/libero/article-store/pull/47/files#r352084039)